src/wr_logic_ai/nodes/finder.py

```diff
@@ -77,58 +91,76 @@ def get_valley(
-    obstacle_list = list() # format: [[left bound index , right bound index], ...]
-    one_obstacle = [] #this will be an array of two values with the beginning and ending index of the obstacle.
+    # format: [[left bound index , right bound index], ...]
+    obstacle_list = list()
+    # this will be an array of two values with the beginning and ending index of the obstacle.
+    one_obstacle = []
     for i in range(len(hist)):
-        if(hist[i] < threshold):  
+        if(hist[i] < threshold):
             one_obstacle.append(i)
-        elif (len(one_obstacle) != 0):
+        elif (len(one_obstacle) > 1):
             left_bound = len(hist)
             right_bound = 0
             for i in range(len(one_obstacle)):
                 # Calculate size of anti-window and add to obstacle bounds
-                angleToIncrease = calculate_anti_window(hist[one_obstacle[i]]) / sector_angle #pass in distance to target to caculate angle that allows robot to pass through
-                
+                # pass in distance to target to caculate angle that allows robot to pass through
+                angleToIncrease = calculate_anti_window(
+                    hist[one_obstacle[i]]) / sector_angle
+                print(f"angle to increase: {angleToIncrease}")
+
                 # Update left and right bound
-                left_bound = max(min(left_bound, one_obstacle[i]-angleToIncrease), 0)
-                right_bound = min(max(right_bound, one_obstacle[i]+angleToIncrease), len(hist))
-            
+                left_bound = max(
+                    min(left_bound, one_obstacle[i]-angleToIncrease), 0)
+                right_bound = min(
+                    max(right_bound, one_obstacle[i]+angleToIncrease), len(hist))
+
             # Check to see if the obstacle we just found can actually be merged with a previous obstacle
             while len(obstacle_list) > 0 and obstacle_list[-1][1] >= left_bound:
                 left_bound = min(left_bound, obstacle_list[-1][0])
                 right_bound = max(right_bound, obstacle_list[-1][1])
-                del obstacle_list[-1]            
+                del obstacle_list[-1]
             obstacle_list.append([left_bound, right_bound])
+            print("one_obstacle")
+            print(one_obstacle)
             one_obstacle.clear()
-            
+
     if (len(one_obstacle) != 0):
         left_bound = len(hist)
         right_bound = 0
         for i in range(len(one_obstacle)):
             # Calculate size of anti-window and add to obstacle bounds
-            angleToIncrease = calculate_anti_window(hist[one_obstacle[i]]) / sector_angle #pass in distance to target to caculate angle that allows robot to pass through
-            
+            # pass in distance to target to caculate angle that allows robot to pass through
+            # angleToIncrease = calculate_anti_window(hist[one_obstacle[i]]) / sector_angle
+            angleToIncrease = 0
```

Comment:
Restore original behavior for fenceposting:
```suggestion
            angleToIncrease = calculate_anti_window(hist[one_obstacle[i]]) / sector_angle
```